Misc/NEWS.d/next/Library/2020-05-15-19-53-18.bpo-37630.O5kgAw.rst

```diff
@@ -0,0 +1,2 @@
+The :mod:`hashlib` module can now use SHA3 hashes and SHAKE XOF from OpenSSL
+1.1.1 by default.
```

Comment:
I'd word this "will now use X and Y from Z when available" rather than mentioning the version number and by default.